src/router/index.js

```diff
@@ -16,10 +17,20 @@ const router = createRouter({
       component: UserView
     },
     {
-      path: '/login',
+      path: '/',
       name: 'login',
       component: LoginView
     },
+    {
+      path: '/finalize-registration',
+      name: 'finalize-registration',
+      component: LoginView
+    },
+    {
+      path: '/createAccount',
```

Comment:
please use kebab-case in route paths

```suggestion
      path: '/create-account',
```